Ryan doc update (#576)
pysat/pysat: docs/tutorial_basics.rst

and potentially a ``tag`` and ``inst_id``, consistent with
the desired data to be analyzed, from one the supported instruments.

> For example if you wanted to work with plasma data from the

Reviewer: ```suggestion
For example, if you wanted to work with plasma data from the
```

    plt.ylabel(dmsp.meta['ti', dmsp.meta.name_label] + ' (' +
               dmsp.meta['ti', dmsp.meta.units_label] + ')')

> Note, the np.where may be removed using the convenient access to the
> attached pandas data object.

Reviewer: ```suggestion
Note, np.where may be used to select a subset of data using either the convenient access or standard pandas or xarray selection methods.
```

.. code:: python

   idx, = np.where((dmsp['mlat'] < 5) & (dmsp['mlat'] > -5))

>    dmsp.data = dmsp[idx] = dmsp.data.iloc[idx]

Reviewer: ```suggestion
   dmsp.data = dmsp[idx]
   # Alternatively
   dmsp.data = dmsp.data.iloc[idx]
```
